stable/spinnaker/values.yaml

```diff
@@ -31,6 +32,15 @@ dockerRegistries:
 #   password: '<INSERT YOUR SERVICE ACCOUNT JSON HERE>'
 #   email: 1234@5678.com
 
+# If you don't want to put your passwords into a values file
```

Comment:
Can you add something about this to the README as well?

Reply:
done!